src/task.c

```diff
@@ -798,7 +833,7 @@ task_system_create(platform_heap_id    hid,
          *system = NULL;
          return rc;
       }
-      nbg_threads_created += num_bg_threads[type];
+      nbg_threads_created += (use_bg_threads ? num_bg_threads[type] : 0);
```

Comment:
Now that bg-threads are configured only when both config values are non-zero, old code is a bug.

I had to reconcile like so, above, otherwise, we'd run into an endless sleep loop on L840 below ... waiting endlessly for background threads to startup.

Reply:
L836 has been deleted. NO longer needed.